api/v1/server/oas/transformers/v1/workflow_runs.go

```diff
@@ -49,6 +49,13 @@ func WorkflowRunDataToV1TaskSummary(task *v1.WorkflowRunData) gen.V1TaskSummary
 		taskId = int(*task.TaskId)
 	}
 
+	var stepId uuid.UUID
+	if task.StepId != nil {
+		stepId = uuid.MustParse(sqlchelpers.UUIDToStr(*task.StepId))
+	} else {
+		stepId = uuid.Nil
```

Comment:
i dont think i've ever used this before, is this to prevent the 000-000-... empty uuid case?

Reply:
yeah that was the intention but I might just be doing something wrong 😅